ios/browser/api/brave_rewards/brave_rewards_api.mm

```diff
@@ -137,16 +150,12 @@ @implementation BraveRewardsAPI
 
 - (instancetype)initWithStateStoragePath:(NSString*)path {
   if ((self = [super init])) {
-    _engineTaskRunner = base::ThreadPool::CreateSingleThreadTaskRunner(
-        {base::MayBlock(), base::WithBaseSyncPrimitives(),
-         base::TaskPriority::USER_VISIBLE,
-         base::TaskShutdownBehavior::BLOCK_SHUTDOWN},
-        base::SingleThreadTaskRunnerThreadMode::DEDICATED);
+    _taskRunner = base::SequencedTaskRunner::GetCurrentDefault();
```

Comment:
Do we ever post any tasks that can block or is that not possible due to using mojom bindings now?

Reply:
`_taskRunner` here is just the task runner on which the object and its mojo remote are created. All of the actual work is done on the other sequences associated with each worker.